src/main/java/com/revature/_1811_nov27_wvu/icebox/dao/UserDao.java

```diff
@@ -0,0 +1,18 @@
+package com.revature._1811_nov27_wvu.icebox.dao;
+
+import java.util.List;
+import java.util.Set;
+
+import org.hibernate.Session;
+import org.springframework.stereotype.Component;
+
+import com.revature._1811_nov27_wvu.icebox.entity.User;
+
+@Component
```

Comment:
The `@Component` annotation should be on the implementation and not the interface.